tests/cases/helpers/case_helpers_care_platform.yaml

```diff
@@ -1714,4 +1714,26 @@ CarePartnerEditParticipantNavigation:
       Strategy: xpath
       Id: $PAGE.care_platform_participant_records.edit_info_button$
     - Type: sleep
-      Time: 1
\ No newline at end of file
+      Time: 1
+
+CarePartnerToastifyValidation:
+  Environment: Settings
+  Roles:
+    - Role: desktopChrome
+      App: desktop
+  Actions:
+    - Type: get_text
+      Strategy: xpath
+      Id : $PAGE.care_platform.toastify_first_element_text$
+      Greps:
+        - var: TOASTIFY_MESSAGE
+          match: "(.*)"
+    - Type: assert
+      Role: desktopChrome
+      Asserts:
+        - Type: eq
+          Var: TOASTIFY_MESSAGE
+          Value: $AND_CLI_TOASTIFY_MESSAGE_SHOWN$
+    - Type: wait_not_visible
+      Strategy: xpath
+      Id: $PAGE.care_platform.toastify_first_element_text$
```

Comment:
I'm not sure this is the best way to handle or assert a notification.
Reasons:
-You can have more than one toast at any time, and there's no way to be sure that the first one will be the one with the data you are looking for.
-You are not using an index, so if there's more than one at the time this is running, it would break, elements need a selector that defines only one element. You are using 'toastify_first_element_text' but you are not doing anything to ensure you get the actual first.

Reply:
I change the selector using text instead of class, so I just using a variable to confirm the message in showing in the test.